Automattic/vip-go-mu-plugins
# Add Jetpack 9.3 Beta

## Description

Adds Jetpack 9.3-beta for easy testing.

Also tweaks the `.gitignore` file to omit several unneeded files.

## Changelog Description

### Jetpack 9.3 Beta Added

Jetpack 9.3-beta has been released and is now available in mu-plugins. To test the new version, simply add this constant to your vip-config.php file: define( 'VIP_JETPACK_PINNED_VERSION', '9.3');

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR
2. Set your site to use JP 9.3 beta with `define( 'VIP_JETPACK_PINNED_VERSION', '9.3');` in vip-config.php
3. Visit the JP dashboard and ensure 9.3-beta is the reported version
4. Test various site functionality
